src/modules/appointment/models/appointment.entity.ts

```diff
@@ -306,4 +307,11 @@ export class AppointmentEntity {
   //   // @OneToOne(() => Employees)
   //   // @JoinColumn({ name: 'receptionist_id' })
   //   // receptionist: Employees;
+
+//   @ManyToOne(() => PatientEntity, patient => patient.appointments)
+// patient: PatientEntity;
```

Comment:
I think that this is not necessary the relation in this entity

Reply:
If I do not make this relationship with this entity, I have problems consuming